libretranslate/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3"
+
+services:
+  app_proxy:
+    environment:
+      # <app-id>_<web-container-name>_1
+      # e.g. 'btc-rpc-explorer_web_1'
+      # Note that the '_1' at the end is needed
+      APP_HOST: libretranslate_main_1
+      APP_PORT: 5000
+      PROXY_AUTH_ADD: "false"
```

Comment:
do we want to change this so that the app is protected by the app_proxies inbuilt authentication? Or do you think it's better that an app like this is completely unauthenticated?

Reply:
The main feature of the app is it's api, which would be impossible to use if app auth is enabled.

Reply:
I don't think the self-hosted api supports authentication, so protecting the main page with app-auth would be pointless. I'll check.

Reply:
> The main feature of the app is it's api, which would be impossible to use if app auth is enabled.
> I don't think the self-hosted api supports authentication, so protecting the main page with app-auth would be pointless. I'll check.

We can still enable authentication via the app proxy and then whitelist the api route (e.g., like [invoice ninja](https://github.com/getumbrel/umbrel-apps/blob/098c868b3f2ace8f9fa0db1e2c0efa6dc28c145f/invoice-ninja/docker-compose.yml#L3C1-L8C37)), but I take your point. If we want to enable auth in a future app update it is a simple change.